salt/states/virt.py

```diff
@@ -259,7 +445,6 @@ def running(name,
             install=True,
             pub_key=None,
             priv_key=None,
-            update=False,
```

Comment:
why are you removing this kwarg without properly deprecating it? This would require users to update their states without warning

Reply:
reintroduced the parameter with a deprecation warning.